qcodes/instrument_drivers/tektronix/Keithley_2450.py

```diff
@@ -108,20 +123,24 @@ def __init__(self, name, address, **kwargs):
                            vals=Numbers(),
                            get_cmd=self._get_relative_offset,
                            set_cmd=self._set_relative_offset,
+                           get_parser=float,
+                           set_parser=float,
                            label='Relative offset value for a measurement.',
                            docstring='This specifies an internal offset that can be applied to measured data')
 
         self.add_parameter('relative_offset_state',
-                           val_mapping={'ON': 1, 'OFF': 0},
```

Comment:
It's a tricky situation what would be the best here. I understand the reasoning for 0, 1.
Personally however, my preference goes out to boolean True / False, as it's the reason why the boolean was introduced.
What do you think? 
I see in this file that a lot of different terms are used ('ON'/'OFF', 'YES'/'NO', even 'True'/'False' in string format!)